### PR TITLE
Add K8sConsulPrefix

### PR DESCRIPTION
**Pull request by s-christoff**
Updates values file to allow users to prefix k8s services registered within consul with a string.

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -260,7 +260,7 @@ ui:
 # Consul agent.
 syncCatalog:
   # True if you want to enable the catalog sync. "-" for default.
-  enabled: false
+  enabled: true
```
Comment (adilyse) [inline values.yaml]: This should stay `false` for the default helm chart.

File: values.yaml [hunk 2]
```diff
@@ -274,6 +274,12 @@ syncCatalog:
   # prepended with "consul-". (Consul -> Kubernetes sync)
   k8sPrefix: null
 
+  # consulServicePrefix is the service prefix which preprends itself 
+  # to Kubernetes services registered within Consul
```
Comment (adilyse) [inline values.yaml]: Nit:
```suggestion
  # to Kubernetes services registered within Consul.
```

File: values.yaml [hunk 3]
```diff
@@ -274,6 +274,12 @@ syncCatalog:
...
+  # For example, "k8s-" will register all services peprended with "k8s-". 
+  # (Kubernetes -> Consul sync)
+  consulServicePrefix: null
```
Comment (adilyse) [inline values.yaml]: Does it make sense to name this `consulPrefix` to be consistent with `k8sPrefix`?

File: values.yaml (hunk 1 above)
Comment (s-christoff) [reply values.yaml]: 🤦‍♀️sorry, I thought I fixed this too.

**Pull request merged**